internal/cmd/connect/command_plugin_install.go

```diff
@@ -0,0 +1,456 @@
+package connect
+
+import (
+	"archive/zip"
+	"bytes"
+	"crypto/md5"
+	"crypto/sha1"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/form"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/confluentinc/cli/internal/pkg/utils"
+)
+
+type manifest struct {
+	Owner struct {
+		Username string `json:"username"`
+		Name     string `json:"name"`
+	} `json:"owner"`
+	Name    string `json:"name"`
+	Title   string `json:"title"`
+	Version string `json:"version"`
+	Archive struct {
+		Url  string `json:"url"`
+		Md5  string `json:"md5"`
+		Sha1 string `json:"sha1"`
+	} `json:"archive"`
+	License []struct {
+		Name string `json:"name"`
+		Url  string `json:"url"`
+	} `json:"license"`
+}
+
+const (
+	archiveInstallation = "ARCHIVE"
+	packageInstallation = "PACKAGE"
+
+	invalidDirectoryErrorMsg       = `plugin directory "%s" does not exist`
+	unexpectedInstallationErrorMsg = "unexpected installation type: %s"
+)
+
+func (c *pluginCommand) newInstallCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "install <plugin>",
+		Short: "Install a Connect plugin.",
+		Args:  cobra.ExactArgs(1),
+		RunE:  c.install,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: "Install the latest release of the Datagen connector onto your local Confluent Platform environment.",
+				Code: "confluent connect plugin install confluentinc/kafka-connect-datagen:latest",
+			},
+		),
+		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireOnPremLogin},
+	}
+
+	cmd.Flags().String("plugin-directory", "", "The plugin installation directory.")
+	cmd.Flags().StringSlice("worker-configs", []string{}, "A comma-separated list of paths to one or more Kafka Connect worker configuration files. Each worker file will be updated to load plugins from the plugin directory in addition to any preexisting directories.")
+	cmd.Flags().Bool("dry-run", false, "Simulate an operation without making any changes.")
```

Comment:
Let's make all of our `--dry-run` flags have the same wording?

Reply:
We can do that. Do we want to use this wording here?